crates/tracing-instrumentation/src/prometheus_metrics.rs

```diff
@@ -66,8 +73,8 @@ impl Prometheus {
         self.handle.as_ref()
     }
 
-    pub fn global_labels(&self) -> &[(String, String)] {
-        &self.global_labels
+    pub fn global_labels(&self) -> Vec<String> {
+        self.global_labels.clone()
```

Comment:
Let's just return a reference to `global_labels` here.

Reply:
At lease we need to modify this vector (to add db and cf labels), so i make a clone at here. But i agree with you, if global_labels is reused at anywhere, return a reference maybe better choice.